app/Casts/AddressCast.php

```
@@ -0,0 +1,50 @@
+        return new Address(
+            street: $attributes['street'] ?? null,
+            city: $attributes['city'] ?? null,
+            province: $attributes['province'] ?? null,
+            zipcode: $attributes['zipcode'] ?? null,
+            country_id: $attributes['country_id'] ?? null,
+            locality: $attributes['locality'] ?? null,
+            address_extra: $attributes['address_extra'] ?? null,
+        );
+    }
+
+    public function set(Model $model, string $key, mixed $value, array $attributes): array
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$model'. (undefined)

(UnusedFormalParameter)